test/sql/create/create_database.test

```diff
@@ -7,9 +7,14 @@ require noforcestorage
 statement error
 CREATE DATABASE mydb;
 ----
-CREATE DATABASE not supported in DuckDB yet
```

Comment:
I updated the test cases here to ensure that these syntax will fail through to the extension but I can also see you may not want these tests anymore so I'm happy to remove this file too. LMK.